TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    coords = SkyCoord(ra,dec,unit='deg')
</pre><p> </p><p>Since we don't actually use the coordinates, I think we should remove this line</p><p> </p><p>(update: I see where we use these below, but I think there's an easier way)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>okay</p>